openapi/index.yaml

```diff
@@ -200,6 +200,9 @@ definitions:
       landingPageReferrer:
         type: string
         minLength: 1
+      discountUrl:
+        type: string
+        minLength: 1
```

Comment:
Can be useful to add maxLength as expressed in backend specs?

Reply:
fixed [f14a77c](https://github.com/pagopa/io-functions-cgn-operator-search/pull/47/commits/f14a77c0e74fc327e833dfb90f7cb82460d526a9)